test/e2e/framework/framework.go

```diff
@@ -168,6 +177,22 @@ func (f *Framework) Helper() *helper.Helper {
 	}
 }
 
+func (f *Framework) CertificateDurationValid(c *v1alpha1.Certificate, duration time.Duration) {
```

Comment:
(regarding rebase): this is fine to keep here, despite other things having moved about 😄